.github/workflows/test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         env:
           TZ: ${{ matrix.tz }}
         run: |
-          PYTEST_ARGS=('-n' '2')
+          PYTEST_ARGS=(-n 2 -m 'slow or not slow')
```

Comment:
Isn't `slow or not slow` equivalent to not specifying any marks?

Reply:
No because of https://github.com/metomi/isodatetime/blob/19929352802aa4c4fe9a72a0475c410775dc58bc/pytest.ini#L22